server/routes/applications.js

```diff
@@ -302,8 +302,12 @@ router.get('/manage-applications/:appId/:keyType/:keyId/update/:env', (req, res,
     .then(keyData => {
       viewData.this_data.keyName = keyData.name;
       viewData.this_data.keyDescription = keyData.description;
-      viewData.this_data.restrictedIps = keyData.restrictedIPs;
-      viewData.this_data.javaScriptDomains = keyData.jsDomains;
+      if(viewData.this_data.keyType === 'key'){
+        viewData.this_data.restrictedIps = keyData.restrictedIPs;
+        viewData.this_data.javaScriptDomains = keyData.jsDomains;
+      }else if(viewData.this_data.keyType === 'web'){
+        viewData.this_data.redirectURIs = keyData.redirectURIs;
```

Comment:
I think this may conflict with mine but incase it hasn't flagged this up, this can just be (keyType === 'key') etc because the keyType is passed as a parameter earlier on